Website/Hotcakes.Views.csproj

```diff
@@ -1563,6 +1562,12 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\Libraries\Hotcakes.Payment\Hotcakes.Payment.csproj">
```

Comment:
Is this reference absolutely necessary?  It wasn't necessary before.  I just want to be careful that we avoid accidentally introducing any circular dependencies.  :)

Reply:
This is used for allowing constants to be used in the primary views.